ember-cli-build.js

```diff
@@ -131,10 +121,9 @@ module.exports = function(defaults) {
     return "'" + item + "'";
   }).join(',') + ']';
 
-  var startupWrapper = pickFiles('ember_debug', {
-    srcDir: '/vendor',
+  var startupWrapper = new Funnel('ember_debug', {
```

Comment:
We can't use the variable since it excludes `vendor/startup-wrapper.js` which we're picking here.